templates/education/pdce-workshops.php

```diff
@@ -11,16 +11,21 @@
 
 $the_title          = get_the_title();
 
-// Some custom fields
-$coursecodeundergrad			= get_post_meta( absint( $post_id ), 'coursecodeundergrad', true );
-$coursecodegrad					= get_post_meta( absint( $post_id ), 'coursecodegrad', true );
-$vanityurl 						= get_post_meta( absint( $post_id ), 'vanityurl', true );
-$credit 						= get_post_meta( absint( $post_id ), 'credit', true );
-$promophoto 					= get_post_meta( absint( $post_id ), 'promophoto', true );
-$subtitle						= get_post_meta( absint( $post_id ), 'subtitle', true );
-$dates							= get_post_meta( absint( $post_id ), 'dates', true );
-$location						= get_post_meta( absint( $post_id ), 'location', true );
+//Get the post permalink
+
+$permalink          = get_permalink( $post );
 
+// Some custom fields
```

Comment:
You're doing `absint( $post_id )` approximately a gerzillion times. It's great that you're sanitizing the variable, but it may be best to do it once and then use that, i.e.

`$post_id = absint( $post_id );`

Above all of the times you use it. It's a micro optimisation and not normally worth worrying about, but in this case it'll also make the code a bit cleaner and easier to read, so imho, worth it. What do you think?